code/game/machinery/lightswitch.dm

```diff
@@ -10,6 +10,23 @@
 	/// instead of the switch's location.
 	var/area/area = null
 
+/obj/machinery/light_switch/directional/north //Dir excluded because tiny switch isn't directional.
+	pixel_y = 27
+	pixel_x = 7
+
+/obj/machinery/light_switch/directional/south
+	pixel_y = -27
+	pixel_x = -7
+
+/obj/machinery/light_switch/directional/east
+	pixel_x = 27
+	pixel_y = 7
+
+/obj/machinery/light_switch/directional/west
+	pixel_x = -27
+	pixel_y = -7
+
+/obj/machinery/light_switch/Initialize()
 /obj/machinery/light_switch/Initialize(mapload)
```

Comment:
🤔

Reply:
That must've changed when I made directional versions of light switches. I'll remove that line shortly.